tests/4-mlut/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# TODO: Add support for eblif based equivilence tests.
```

Comment:
Hrm? I thought you had fixed that?

Reply:
It's a CMake TODO issue.  I removed the logic that was present in the Makefile system to ingest eblif as input.  Need to think about how to add it back.   I was thinking about refactoring the synthesis from the place and route.